portfolio/src/main/webapp/index.html

```diff
@@ -12,6 +12,9 @@
     </head>
 
     <body bgcolor="#e2eafc"> <!-- bgcolor="#ffc4d6" #fae0e4-->
+        <h1>Servlet Try</h1>
+        <p>Click <a href="/random-servlet">here</a> to view a random phrase generated by a servlet.</p>
```

Comment:
This `/random-servlet` URL doesn't seem to line up with other changes in your PR. Are you sure this is the right URL? My inclination would be to think that this path and the one in the `@WebServlet()` should match.